libgiza/git.py

```diff
@@ -202,6 +202,25 @@ def update(self):
     def pull(self, remote='origin', branch='master'):
         return self.cmd('pull', remote, branch)
 
+    def push(self, remote='origin', ref=None):
+        if ref is None:
+            ref = self.current_branch()
+
+        return self.cmd("push", remote, ref)
+
+    def tag(self, name, delete=False, force=False):
+        args = ["tag"]
+
+        if delete is True:
```

Comment:
bleh, I remembered this admonition but in reverse, but it's all over giza/libgiza. maybe I'll change it in a larger sweep later?